impeller/docs/blending.md

```diff
@@ -0,0 +1,95 @@
+# Color blending
+
+Impeller currently supports the same set of blending operations that
+[Skia](https://api.skia.org/SkBlendMode_8h.html#ad96d76accb8ff5f3eafa29b91f7a25f0)
+supports. Internally, Impeller distinguishes between two different kinds of
+blend modes: Those which can be performed using the raster pipeline blend
+configuration (called "Pipeline Blends"), and those which cannot (called
+"Advanced Blends").
+
+All blend modes conform to the
+[W3C Compositing and Blending recommendation](https://www.w3.org/TR/compositing-1/).
+
+Blend operations are driven by the `Entity::BlendMode` enum. In the Aiks layer,
+all drawing operations conform to the given `Paint::blend_mode`. In the Entities
+layer, all Entities have an associated blend mode, which can be set via
+`Entity::SetBlendMode(Entity::BlendMode)`.
+
+## Glossary of blending terms
```

Comment:
In a later patch perhaps, the performance implications of advanced blends could be added.

Reply:
Added some details about why advanced blends are much slower than pipeline blends.